posts/Yoann/2017-11-27-introducing-hibernate-search-jsr352-mass-indexing-job.adoc

```diff
@@ -0,0 +1,163 @@
+= Introducing the Hibernate Search JSR-352 mass indexing job
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "JSR-352" ]
+:awestruct-layout: blog-post
+---
+
+Hibernate Search 5.9 will feature integration with JSR-352 runtimes,
+providing a new implementation of mass indexing (indexing a high volume of entities)
+as a JSR-352 job.
+
+Today we released 5.9.0.Beta1 so you can try it out already!
+
+Why is it nice? How to use it? Let's find out!
+
+== What's JSR-352?
+
+"JSR 352: Batch Applications for the Java Platform" is a Java EE specification
```

Comment:
It's not targeting Jave EE exclusively.

Reply:
damn you're essentially suggesting the opposite of what I suggested before :D 
 - https://github.com/hibernate/in.relation.to/pull/98#discussion_r153006248

he's not saying that it's exclusive to Java EE, yet it is a Java EE specification.

Maybe just "specification" to make everyone happy? or ignore us Yoann ;)

Reply:
Just "specification" is good. I'd equally cringe if you said BV is a Java EE specification. Yes, it is part of EE, but it also runs under SE. Saying it's a EE spec obfuscates this fact.